test/kbaseauthcontroller/controller.go

```diff
@@ -48,7 +50,11 @@ type Controller struct {
 
 // New creates a new controller.
 func New(p Params) (*Controller, error) {
-	classPath, err := getClassPath(p.JarsDir)
+	classPath, err := getClassPath(p.Auth2Jar)
+	if err != nil {
+		return nil, err
+	}
+	tpath, err := pullTemplatesOutofAuth2Jar(classPath)
```

Comment:
I would combine the functionality of pullTemplates... and installTemplates. Just take the jar and target template directory as inputs and install the templates there

Reply:
In my last commit, pullTemplates was inside installTemplates. Why you think is better this way?

Reply:
I'm not sure what you mean - the last time I reviewed installTemplates wasn't modified at all. There was a small change to getClassPath but that's it
https://github.com/kbase/blobstore/pull/141/files/95e7f5c41fd186ab3585d10179c821647af4e184

Reply:
But anyway, I think merging the methods is better because there's no reason to separate them. What we want is the templates removed from the jar file and put in a templates directory, which can be done in a single method. We don't need the intermediate state of the unzipped jar (in fact, we don't need to unzip the jar contents at all if we can just read the templates dir and pull them out individually

Reply:
Got it. Last commit: https://github.com/kbase/blobstore/pull/141/commits/8576aab0b7facf660a854bc050046ac18c25e358

Reply:
Oh, I never reviewed that one so I never new it was originally inside installTemplates

Reply:
👍

Reply:
:+1: